Dockerfile

```diff
@@ -157,7 +157,8 @@ RUN /opt/ort/bin/import_proxy_certs.sh && \
 
 # Add scanners (in versions known to work).
 ARG SCANCODE_VERSION
-RUN pip install --no-cache-dir scancode-toolkit==$SCANCODE_VERSION
+RUN wget https://raw.githubusercontent.com/nexB/scancode-toolkit/v$SCANCODE_VERSION/requirements.txt
```

Comment:
Nit: We try to stick to `curl` in this file, so please probably use `curl -Os` instead.

Reply:
And please combine this into the below `RUN` statement to create fewer layers. Like in total

```
RUN curl -Os https://raw.githubusercontent.com/nexB/scancode-toolkit/v$SCANCODE_VERSION/requirements.txt && \
    pip install --no-cache-dir scancode-toolkit==$SCANCODE_VERSION
```